### PR TITLE
Add autoprefixer

### PR DESCRIPTION
**Pull request by ezhlobo**
- [x] https://trello.com/c/mKAYw3dA/190-get-rid-of-prefixes-in-css-use-autoprefixer

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://add-autoprefixer.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://add-autoprefixer.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: browserslist [hunk 1]
```diff
@@ -0,0 +1 @@
+last 2 versions
```
Comment (Proghat) [inline browserslist]: What do you think about adding more versions? More than 1%, for example.

datarockets.com is public site.

Comment (ezhlobo) [inline browserslist]: @Proghat suggest to analyse statistic of visitors. I'll do it. Thanks for paying attention.

Comment (Proghat) [inline browserslist]: Thank you!

Comment (Proghat) [inline browserslist]: @ezhlobo, what do you think about >1% in world?

**Comment by evgenTraytyak** [comment]
#https://github.com/datarockets/datarockets.com/pull/10

File: browserslist (hunk 1 above)
Comment (ezhlobo) [inline browserslist]: @Proghat done ;)

**Comment by evgenTraytyak** [comment]
:shipit:

**Pull request merged**